core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.template.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-header">
+  <h3>
+    Create New Subtopic
+  </h3>
+</div>
+<div>
+  <div class="modal-body new-subtopic-editor">
+    <p>
+      Enter the title for the subtopic
+    </p>
+    <input class="form-control protractor-test-new-subtopic-title-field" style="font-size: 12px;"
+           placeholder="Enter a brief description for the subtopic that students can easily understand, e.g. 'Adding Fractions'"
+           ng-model="$ctrl.subtopicTitle" ng-change="$ctrl.resetErrorMsg()" maxlength="<[MAX_CHARS_IN_SUBTOPIC_TITLE]>" autofocus>
+    <span class="oppia-input-box-subtitle">
+      <i>
+        Subtopic title should be at most <[MAX_CHARS_IN_SUBTOPIC_TITLE]> characters.
+      </i>
+    </span>
+    <span class="form-text" style="font-size: smaller; color: red;" ng-if="$ctrl.errorMsg">
+      <em><[errorMsg]></em>
+    </span>
+
+    <div class="thumbnail-editor">
+      <thumbnail-uploader filename="$ctrl.editableThumbnailFilename"
+                          update-filename="$ctrl.updateSubtopicThumbnailFilename"
+                          bg-color="$ctrl.editableThumbnailBgColor"
+                          update-bg-color="$ctrl.updateSubtopicThumbnailBgColor"
+                          allowed-bg-colors="$ctrl.allowedBgColors"
+                          aspect-ratio="'4:3'"
+                          preview-title="$ctrl.editableTitle"
+                          preview-description=""
+                          preview-description-bg-color="'#BE563C'"
+                          preview-footer="">
+      </thumbnail-uploader>
+    </div>
+    <div class="description-editor" ng-if="$ctrl.schemaEditorIsShown">
+      <div class="oppia-editor-card-body" >
+        <schema-based-editor schema="$ctrl.SUBTOPIC_PAGE_SCHEMA"
+                             class="protractor-test-create-subtopic-page-content"
+                             local-value="$ctrl.htmlData">
+        </schema-based-editor>
+      </div>
+    </div>
+
+    <div ng-if="!$ctrl.schemaEditorIsShown" ng-click="$ctrl.showSchemaEditor()" class="protractor-test-show-schema-editor">
```

Comment:
Add a title above the subtopic page contents editor, currently it looks weird with no title:
![image](https://user-images.githubusercontent.com/22347970/85408209-d9efc700-b581-11ea-974a-ea6c68a69ca7.png)

Reply:
Fixed.